app/_locales/en/messages.json

```
@@ -596,6 +596,10 @@
   "confirm": {
     "message": "Confirm"
   },
+  "confirmPageDialogSetApprovalForAll": {
+    "message": "You’re allowing access to all of your $1 NFTs plus any future NFTs in this collection. This should never be your default, no matter how secure a site may seem. Proceed with caution.",
```

I believe our general content guidelines are to use punctuation for any sentences that have proceeding sentences but not at the end of the last sentence? @coreyjanssen can you confirm?

```suggestion
    "message": "You’re allowing access to all of your $1 NFTs plus any future NFTs in this collection. This should never be your default, no matter how secure a site may seem. Proceed with caution",
```